contracts/NFT/ERC721UpgradableAirdroppable.sol

```diff
@@ -46,4 +56,9 @@ abstract contract ERC721UpgradableAirdroppable is ERC721UpgradableBase, IAirdrop
     $._nextTokenId = tokenId;
     emit AirdropExecuted(receivers.length);
   }
+
+  function lockNFTMinting() external onlyOwner {
+    _getStorage()._locked = true;
+    emit AirDropLocked(totalSupply());
+  }
 }
```

Comment:
You are locking it against your own temptation to do a couple of more mints? 🤣 funny 
And later you say to yourself: oh thank goodness, I was wise that locked it earlier! Now I wouldn't be able to resist